docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 services:
 
   db:
-    image: postgres:12
+    image: postgres:16
```

Comment:
Great! :rocket:

Reply:
It seems that Postgres has officially released version `17` four days ago. Should I stick to `16` or upgrade it to `17`?

Reply:
Since you were working on more importatn tasks (perfect SQLModel Documentation, by the way!😉), I changed it to `17` without your permission! Sorry for that. If you still wish to use `16`, just tell me to change it.